### ISSUE TITLE
Error 500 on php page

### ISSUE THREAD

**Issue by viperzero** [hestiacp/hestiacp#319]
### Brief issue description:
I get an error 500 when trying to access subfolder/index.php
 error log gives following errors:

 /var/www/html/zc_install/.htaccess: RewriteRule not allowed here, referer: http://94.xxx.xxx.13/index.php

/var/www/index.html shows up fine

I don't have domain name attached, just using ip

### Operating system distribution and release:
. Ubuntu 18.04 LTS

### HestiaCP Version:
Type here (e.g. 0.9.8-28)

### Which branch are you using?
Master

### What software is installed?
Apache,, Dovecot/Exim, MariaDB

**Comment by ScIT-Raphael** [comment]
Thanks for your bug report, but due to your file paths (/var/www), it looks like this has nothing to do with hestia. Hestia use the directories in /home/user/web/domain.tld/public_html. Can you write more about how you reproduce the issue? Also add as much details as possible and check also the log files.

**Comment by viperzero** [comment]
Hi,
sorry. I am totally new to this control panel and I tried to make an forum account but it was not confirmed with email.

I solved the problem. It was simple. My hosting provider uses both public and private ip and the dns settings were pointing to private ip. After I changed them to public and uploaded the files to correct path everything is starting to work.

I really like that Hestia supports Ubuntu 18.04LTS and seems very functional all ready. So this was kust a silly beginners mistake and I hope you open the forum soon. Good luck and God speed with development!
